src/draw/DefaultDrawer.js

```diff
@@ -122,6 +122,9 @@ class DefaultDrawer {
         state: false,
         target: null,
       },
+      zoom: {
```

Comment:
documentation ?

Reply:
we have never documented the value of the actions